### PR TITLE
[DO NOT MERGE] - First up for the dataviz dashboard

### PR DESCRIPTION
**Pull request by jeremiak**
Adds a new /dashboard route and experiments with the britechart.js library and a quick bar chart. Not perfect or even done, just pushing up to get feedback from the rest of the team.

This is a duplicate of #100, which I'm going to close but is hard to manage since its from a fork and we've got multiple folks working on this work stream.

### CODE REVIEW

**Comment by kevee** [comment]
Hi, @jeremiak, all this work looks incredible, and I'm OK with it getting merged. What else do you think needs to be done to get this live?

**Comment by jeremiak** [comment]
@kevee Give us a couple of hours? We're sprucing up the page a bit, making all the design consistent, and we're going to need to get the #website channel folks to give it a read through before we merge.

File: _src/_assets/js/dashboard-charts.js [hunk 1]
```diff
@@ -104,10 +105,27 @@
         value: d.positiveIncrease + d.negativeIncrease,
       })),
     ].slice(0, cdcData.length)
-    // debugger
     const cdcChartContainer = d3.select('#cdc-test-chart')
     const ctChartContainer = d3.select('#ct-test-chart')
     const yMax = d3.max(ctData, d => d.value)
+    const cdcSpeciumSum = d3.sum(transformedCdcData, d => {
```
Comment (goleary) [inline _src/_assets/js/dashboard-charts.js]: slightly more concise way to write this:
`const cdcSpeciumSum = d3.sum(transformedCdcData, d => +d.value)`

File: _src/_assets/js/dashboard-charts.js [hunk 2]
```diff
@@ -104,10 +105,27 @@
...
+      return +d.value
+    })
+    const ctTotalTestSum = d3.sum(rawCtData, d => {
```
Comment (goleary) [inline _src/_assets/js/dashboard-charts.js]: &
`const ctTotalTestSum = d3.sum(rawCtData, d => d.negative + d.positive)`

**Comment by kevee** [comment]
Hi, @jeremiak! Do you think this work is stable enough for us to do the work of getting it into the new website? I think it won't be too difficult, but we can start by doing a separate branch off this one to work from.

**Comment by jeremiak** [comment]
@kevee We for sure can! @goleary and I are doing that now, well he's getting a move on it while I try and debug some build issues I'm having with the website. But yes, we are on the same page, and we'll either open a new PR or force push this one to change it so that its using the new gatsby site

**Comment by goleary** [comment]
closing as we are porting dashboard to a gatsby based branch.

**Pull request closed without merging**